### DISCUSSION TITLE
Welcome to CashFactory Discussions!

### DISCUSSION THREAD

**Discussion by OlivierGaland** [OlivierGaland/CashFactory#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OlivierGaland** [comment]
I confirm that the software does not work on a Raspberry (tried and got many errors regarding apps, suspecting most docker images are not supporting arm architecture).

**Comment by OlivierGaland** [comment]
16 March, 2022

Please note that peer2profit website is not accessible on my side (i'm located in France), I suspect this is linked to current event (war in ukraine).

It seems the client is still sharing data even if you can't access the website.
I managed to access the website using a VPN (with identity : Israel, but others countries could work)

[Edit] : solved

**Comment by OlivierGaland** [comment]
19 March, 2022

I've found another passive earning application of bandwidth sharing called Traffmonetizer, it is currently under investigation to see if it works correctly (I've registered and account and it is running).

Unfortunately it is available only on Windows, Android or MacOS, so I cannot integrate it in cashfactory, maybe later if it develops and release a linux compatible.

Link below (with affiliate) :
[Traffmonetizer](https://traffmonetizer.com/?aff=52057)

Olivier

![image](https://user-images.githubusercontent.com/26048157/159097494-8be2e295-4a15-4fa3-a7ef-4bd00bdb4a5e.png)

**Comment by Micka-R** [comment]
Hello,
I am an IT student in France, I'm using CashFactory to see if it could let me gain enough to cover the costs for my self-hosted server upgrades (a new hhd/a raspberry ect...)

I mainly developed knowledge about server hosting/management, but I began learning node JS for some personal project and with some friends we are working on a social network with C# for our 2nd semester.

I find the project interesting, I will help it develop if I can.

**Reply by OlivierGaland** [reply]
Thanks,

Many thanks, the project is up to accept any contribution, I started it for myself when i got interested by passive income app and found it may be interesting for other people.

Olivier.

**Comment by nitrup** [comment]
Hello,
I am a beginner in linux and docker, how to duplicate existing containers?
